pennylane_lightning/src/Gates.hpp

```diff
@@ -20,6 +20,7 @@
 #include <memory>
 #include <vector>
 
+#include "StateVector.hpp"
```

Comment:
@ThomasLoke suspecting that the tests fail on `pybind` because of this include and we'd now need to use it when compiling `Gates.cpp` for the tests

Reply:
Yeah, but I think this becomes a non-issue when #86 is landed.